### PR TITLE
Enable Django admin

### PR DESCRIPTION
**Pull request by mvandenburgh**
Adds the needed settings in order to enable the Django admin at `/admin`. I sometimes find this easier to use then `shell_plus` or looking at the database itself.

Settings needed to enable this were taken from
https://github.com/girder/django-composed-configuration

### CODE REVIEW

**Comment by mvandenburgh** [comment]
Note this will not expose the admin on the AWS deployment, as NGINX only proxies backend requests to `/api`, and not `/admin`.

**Comment by BryonLewis** [comment]
Am I missing something in this to get actual template rendering so the admin interface isn't just raw ugly HTML without styling?

File: django/src/rdwatch/server/settings.py [hunk 1]
```diff
@@ -21,12 +21,19 @@ class BaseConfiguration(Configuration):
     WSGI_APPLICATION = 'rdwatch.server.application'
     ALLOWED_HOSTS = ['*']
     DEBUG = values.BooleanValue(False, _environ_prefix='RDWATCH_DJANGO')
```
Comment (BryonLewis) [inline django/src/rdwatch/server/settings.py:23]: Not sure what is going on, but this isn't evaluating to True when RDWATCH_DJANGO_DEBUG is set to 1 on my system.  Looking at all os.environ I can see it in there, but this is always returning false;
```suggestion
    DEBUG = 'RDWATCH_DJANGO_DEBUG' in os.environ
```

**Review by BryonLewis — CHANGES_REQUESTED** [review]
I like the idea of having this in there to look at stuff quickly instead of utilizing pgadmin or the database directly.

One additional suggestion:
Adding to the docs some instructions about createsuperuser for the admin user docker: `docker compose run --rm django poetry run django-admin createsuperuser`  and maybe the regular command for non-docker dev.

File: django/src/rdwatch/server/settings.py (hunk 1 above)
Comment (mvandenburgh) [reply django/src/rdwatch/server/settings.py:23]: My bad, I fixed this locally but didn't push it here - I opened a PR on top of this branch with the fix #236 as its unrelated to this PR.

**Comment by mvandenburgh** [comment]
Added instructions for `createsuperuser` to the dev docs.

File: django/src/rdwatch/server/settings.py (hunk 1 above)
Comment (BryonLewis) [reply django/src/rdwatch/server/settings.py:23]: Marking as resolved, your other PR clears it up.

**Review by BryonLewis — APPROVED** [review]

**Pull request merged**